### ISSUE TITLE
Issues in Firefox

### ISSUE THREAD

**Issue by BrontoRed** [Pitmairen/selection-search#98]
Some issues present only in Firefox:
1. Popup Menu doesn't work in input text area
2. Popup Menu on http://rutor.info is positioned far away from the mouse cursor ("Position the button relative to the mouse cursor" has no effect)
3. no icons in Context Menu ([Swift Selection Search](https://addons.mozilla.org/firefox/addon/swift-selection-search/) supports it)

**Comment by BrontoRed** [comment]
One more issue present only in Firefox:
search from Context Menu doesn't work in windows without toolbars.
Some sites and extensions (e.g. "[New window without toolbar](https://addons.mozilla.org/firefox/addon/new-window-without-toolbar/)" addon) allow to open any tab in a window without toolbars. But searching from Context Menu in such windows doesn't open new tab with search results. 
At the same time, searching from Popup Menu opens new tab in main Firefox window with search results.

**Comment by hien4990** [comment]
yeah i have those issues too, waited for 2 years with no fix, especially the input text area issues.

**Comment by famicom9x** [comment]
not only selection search most firefox addon doesnt work with input text area, still dont know why?

**Comment by eMPee584** [comment]
Ok here's a working implementation: https://github.com/FilipePS/Traduzir-paginas-web/blob/master/src/contentScript/translateSelected.js
It has several handlers `onKeyUp()`, `onMouseup()`, `onTouchend()` and `onSelectionchange()` that also take special care of ´<input>` and `<textarea>` elements..

**Comment by eMPee584** [comment]
`object.addEventListener("select", myScript);` is suggested as approach on the stack overflow question [Reacting to selection changes in an HTML textarea](https://stackoverflow.com/a/55173327/200509)

**Comment by eMPee584** [comment]
Just to update that `selection-search` now does work in input boxes and text areas, thanks a lot! 🎉

(The issue with no-toolbar windows seems still present, but it is very niche and should probably have a separate issue and, for the sake of project management, this one could be renamed to be solely about the icons in context menu.. @BrontoRed)

**Comment by Pitmairen** [comment]
I kind of forgot about the icons, but I did a quick test now and the icons seems to work fine in the context menu with Firefox. 

I will include the context menu icons in the next version.

**Comment by Pitmairen** [comment]
Icons in the context menu has been added in the latest release.

Also fixed an issue where the items in the context menu could get a bit messed up after updating the settings.

**Comment by eMPee584** [comment]
icons look really good, kudos for coming back to this 👍
